Documentation/data-collection.md

```diff
@@ -594,6 +594,21 @@ data:
     #
     # The current amount of CPU used by Application Services products, aggregated by product name.
     - '{__name__="appsvcs:cores_by_product:sum"}'
+    #
+    # owners: (@openshift/team-build-api)
+    #
+    # openshift:openshift_csi_share_configmap_total:sum measures total amount of config maps shared by csi shared resource driver.
+    - '{__name__="openshift_csi_share_configmap_total"}'
+    #
+    # owners: (@openshift/team-build-api)
+    #
+    # openshift:openshift_csi_share_configmap_total:sum measures total amount of secrets shared by csi shared resource driver.
+    - '{__name__="openshift_csi_share_secret_total"}'
+    #
+    # owners: (@openshift/team-build-api)
+    #
+    # openshift:openshift_csi_share_mount_total:sum measures total amount of csi shared resources mounted in the pods.
+    - '{__name__="openshift_csi_share_mount_total"}'
```

Comment:
about this [metric](https://github.com/openshift/csi-driver-shared-resource/blob/32fffa0df52e9be11b8f8646fadbc4f48a11c08b/pkg/metrics/metrics.go#L24-L30), it's usually better to track requests and failures in different counters (see https://promcon.io/2017-munich/slides/best-practices-and-beastly-pitfalls.pdf, page 10). For instance, `openshift_csi_share_mount_requests_total` and `openshift_csi_share_mount_failures_total`.

Reply:
I don't see this recommendation to use different metrics instead of different labels in the slides. Slides do suggest that it's better to use `failure_total` and `requests_total` instead of `failure_total` and `success_total` though. Do you think we should go all the way back to already merged metrics and change them again?
@gabemontero  wdyt?

Reply:
Again i would recommend that you follow the recommendations. The idea of tracking requests and failures is that you can later alert on the percentage of operations failing.

Reply:
This was discussed on the [PR introducing this](https://github.com/openshift/csi-driver-shared-resource/pull/54/files#r735575449) and I recommended the label way of solving this.

Calculating error rates should still be quite convenient with this setup `rate(openshift_csi_share_mount_failures_total{succeeded="false"}[5m]) / rate(openshift_csi_share_mount_failures_total[5m])`, though admittedly not as short as following the best practice.

@alicerum Since I approved the current state, I'd be happy to propose the change in the exporter.

Reply:
@jan--f After discussion with colleagues we decided it's better to go along with best practices, so i'm going to split them into `total` and `failed` series without labels. Thank you!

Reply:
See https://github.com/openshift/csi-driver-shared-resource/pull/78 if you prefer to go with @simonpasquier's recommendation.